### PR TITLE
fix: URL crash

### PR DESCRIPTION
**Pull request by hayden-muzz**

### CODE REVIEW

File: Ocarina/OcarinaManager.swift [hunk 1]
```diff
@@ -238,8 +238,13 @@ extension OcarinaManager: URLSessionDataDelegate {
         }
         
         var html: HTMLDocument? = nil
+
         if let data = data {
-            html = try? HTML(html: data, encoding: .utf8)
+            do {
+                html = try HTML(html: data, encoding: .utf8)
+            } catch {
+                self.completeRequestsWithError(error, for: originalURL)
```
Comment (newboadki) [inline Ocarina/OcarinaManager.swift]: It seems like we could call `completeRequestsWithError` twice now, here and below in line 255. Is that correct?

File: Ocarina/OcarinaManager.swift [hunk 2]
```diff
@@ -238,8 +238,13 @@ extension OcarinaManager: URLSessionDataDelegate {
         }
         
         var html: HTMLDocument? = nil
+
         if let data = data {
```
Comment (newboadki) [inline Ocarina/OcarinaManager.swift:242]: `If let data {...}` should be enough or does this need to support an older swift version?

**Review by newboadki — CHANGES_REQUESTED** [review]
Small concern about potentially calling a method twice now.

File: Ocarina/OcarinaManager.swift (hunk 1 above)
Comment (hayden-muzz) [reply Ocarina/OcarinaManager.swift]: Yeah tbf I don't need any of these changes. I'll revert it, it was test code but I just left it. Will revert and just have the version bump only

File: Ocarina/OcarinaManager.swift (hunk 2 above)
Comment (hayden-muzz) [reply Ocarina/OcarinaManager.swift:242]: I'm not sure, but I can't actually compile the forked package in Xcode. So i'd prefer as minimal changes as possible. For that reason, will leave as the original code just incase

**Review by newboadki — APPROVED** [review]

**Pull request merged**